bin/Main.cpp

```diff
@@ -48,7 +48,7 @@ inline void run(int argc, char * argv[])
 
 	//Command line input to program is the file that contains the simulation file names
 	string	inputFile;
-	if (argc <= 1) { inputFile = "TestInputs/ParamInput.txt"; }
+	if (argc <= 1) { inputFile = ""; }
```

Comment:
Oh we should just replace this with an error saying you need an input file

Reply:
I could go either way. Currently the output message reads:

```text
Failed to Open 
terminate called after throwing an instance of 'std::__ios_failure'
  what():  basic_ios::clear: iostream error
```

Reply:
Added my preference